plugins/newsdownloader.koplugin/epubdownloadbackend.lua

```diff
@@ -0,0 +1,521 @@
+local Version = require("version")
+local ffiutil = require("ffi/util")
```

Comment:
Having looked at the coding style guide and some of the other lua files, I think some of these should be capitalised (`local FFIUtil = ...`) because they're classes?

Reply:
Either way is fine by me for ffiutil. I think the existing code (most of which predates that addition to the styleguide) may have ffiutil, FFIUtil as well as ffiUtil.

So I'm good to merge, but if you'd like to change it to FFIUtil I'll wait.

Reply:
I'll leave it as it is then :)